Procfile

```diff
@@ -1 +1,2 @@
+release: bundle exec rake db:migrate
```

Comment:
I think rollback means that you deploy an old commit over the current commit. I agree migrating is the wrong approach then. However, I see no other way to solve this currently. I think we should ignore this edge case.